sdk/purview/purview-sharing-rest/review/purview-sharing.api.md

```diff
@@ -157,7 +157,7 @@ export type GetArrayType<T> = T extends Array<infer TData> ? TData : never;
 export function getLongRunningPoller<TResult extends HttpResponse>(client: Client, initialResponse: TResult, options?: CreateHttpPollerOptions<TResult, OperationState<TResult>>): Promise<SimplePollerLike<OperationState<TResult>, TResult>>;
 
 // @public
-export type GetPage<TPage> = (pageLink: string, maxPageSize?: number) => Promise<{
+export type GetPage<TPage> = (pageLink: string) => Promise<{
```

Comment:
This is a breaking, but I feel like since we didn't see a real case of maxPageSize, I feel like it's safe to treat it as a non-breaking ? @xirzec